frameworks/kafka/tests/test_tls.py

```diff
@@ -17,19 +17,15 @@
 @pytest.fixture(scope='module')
 def service_account(configure_security):
     """
-    Creates service account and yields the name.
+    Creates service account for TLS.
```

Comment:
Creates or _updates_?

Reply:
So, since `create_service_account` deletes an existing account -- this may beed to be updated.

What about `"Sets up the service account for the secure tests"`?

Reply:
Ah right. No the original wording is fine...